### PR TITLE
Fix: status manager and update packages

### PR DESCRIPTION
**Pull request by michalpaukert**
### Motivation

Which issue does this fix? Fixes #`issue number`

If no issue exists, what is the fix or new feature? Were there any reasons to fix/implement things that are not obvious?

### Checklist

- [ ] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [x] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

If manual testing is required, what are the steps?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+  "version": "0.0.8",
   "description": "Command line interface tool that can be used for generating and running Kontent migration scripts",
   "main": "./lib/index.js",
   "types": "./lib/types/index.d.ts",
   "scripts": {
     "build": "tsc",
     "test": "jest",
-    "lint": "tslint --project tsconfig.json --fix \"src/**/*.ts\" && prettier --write \"src/**/*.ts\""
+    "lint": "tslint --project tsconfig.json --fix \"src/**/*.ts\" && prettier --write \"src/**/*.ts\"",
+    "debug": "node --inspect ./lib/index.js migration run --all -e DEV"
```
Comment (michalpaukert) [inline package.json:11]: I am not sure if we want to keep this command or not

File: src/utils/statusManager.ts [hunk 2]
```diff
@@ -1,91 +1,85 @@
...
-    status = {};
-};
-
-const updateMigrationStatus = (projectId: string, migrationStatus: IMigrationStatus) => {
-    let projectMigrationsStatus = status[projectId];
-
-    if (projectMigrationsStatus === undefined) {
-        projectMigrationsStatus = status[projectId] = [];
-    } else {
-        const previousMigrationStatus = projectMigrationsStatus.findIndex(migrationStatus => migrationStatus.name === migrationStatus.name);
```
Comment (michalpaukert) [inline src/utils/statusManager.ts:19]: This was a root cause

File: src/utils/migrationUtils.ts [hunk 3]
```diff
@@ -1,181 +1,181 @@
...
+    for (const file of files) {
+        const migrationModule = await loadModule(file.name);
+
+        migrations.push({ name: file.name, module: migrationModule });
+    }
+
+    return migrations.filter(String);
+};
+
+export const getExecutedSuccessMigrations = (migrations: IMigration[], projectId: string): IMigration[] => {
```
Comment (huluvu21) [inline src/utils/migrationUtils.ts]: I would name it _getSuccessfullyExecutedMigrations_.

File: package.json (hunk 1 above)
Comment (huluvu21) [reply package.json:11]: I don't see any issue with keeping it.

**Pull request merged**